### PR TITLE
testing

### PR DESCRIPTION
**Pull request by pjagielski**
Change-Id: Id22af5cab84d12342f4a22e12038dae3b5b32f53

### CODE REVIEW

File: src/main/java/TestFile.java [hunk 1]
```diff
@@ -7,6 +7,7 @@ private static void incorrectAssignmentInIfCondition() {
         boolean value = false;
         if (value = true) {
             //do Something
+            value = false;
```
Comment (pjagielski) [inline src/main/java/TestFile.java]: Nice change

Comment (pjagielski) [inline src/main/java/TestFile.java]: Nice change

File: src/main/java/TestFile.java [hunk 2]
```diff
@@ -7,6 +7,7 @@ private static void incorrectAssignmentInIfCondition() {
...
         } else {
             //else Do Something
```
Comment (pjagielski) [inline src/main/java/TestFile.java]: Nice change 2

File: src/main/java/TestFile.java [hunk 3]
```diff
@@ -7,6 +7,7 @@ private static void incorrectAssignmentInIfCondition() {
...
         }
```
Comment (pjagielski) [inline src/main/java/TestFile.java:13]: Nice change 2

File: src/main/java/TestFile.java [hunk 4]
```diff
@@ -7,8 +7,9 @@ private static void incorrectAssignmentInIfCondition() {
         boolean value = false;
         if (value = true) {
             //do Something
+            value = false;
         } else {
-            //else Do Something
+            //else Do Something 123
```
Comment (pjagielski) [inline src/main/java/TestFile.java]: Nice change 2

Comment (pjagielski) [inline src/main/java/TestFile.java]: Nice change 2

Comment (pjagielski) [inline src/main/java/TestFile.java]: Nice change 2

Comment (pjagielski) [inline src/main/java/TestFile.java]: test

Comment (pjagielski) [inline src/main/java/TestFile.java]: blah blah

Comment (pjagielski) [inline src/main/java/TestFile.java]: blah blah

Comment (pjagielski) [inline src/main/java/TestFile.java]: blah blah

Comment (sputnikci) [inline src/main/java/TestFile.java]: blah blah 123

File: src/main/java/TestFile.java [hunk 5]
```diff
@@ -7,8 +7,9 @@ private static void incorrectAssignmentInIfCondition() {
         boolean value = false;
         if (value = true) {
             //do Something
+            value = false;
```
Comment (sputnikci) [inline src/main/java/TestFile.java]: json

File: src/main/java/TestFile2.java [hunk 6]
```diff
@@ -0,0 +1,13 @@
+class TestFile2 {
```
Comment (sputnikci) [inline src/main/java/TestFile2.java]: [Checkstyle] ERROR: Missing a Javadoc comment.

Comment (sputnikci) [inline src/main/java/TestFile2.java]: [Checkstyle] ERROR: Missing a Javadoc comment.

Comment (sputnikci) [inline src/main/java/TestFile2.java]: [Checkstyle] ERROR: Missing a Javadoc comment.

File: src/main/java/TestFile2.java [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+    public String foo() {
```
Comment (sputnikci) [inline src/main/java/TestFile2.java]: [Checkstyle] ERROR: Missing a Javadoc comment.

File: src/main/java/TestFile2.java [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+        return "bar";
+    }
+
+    private static void incorrectAssignmentInIfCondition() {
```
Comment (sputnikci) [inline src/main/java/TestFile2.java]: [Checkstyle] ERROR: Missing a Javadoc comment.

File: src/main/java/TestFile2.java [hunk 9]
```diff
@@ -0,0 +1,13 @@
...
+        boolean value = false;
+        if (value = true) {
```
Comment (sputnikci) [inline src/main/java/TestFile2.java]: [Checkstyle] ERROR: Inner assignments should be avoided.

File: src/main/java/TestFile2.java (hunk 6 above)
Comment (sputnikci) [inline src/main/java/TestFile2.java]: [Checkstyle] ERROR: Missing a Javadoc comment.

File: src/main/java/TestFile2.java (hunk 7 above)
Comment (sputnikci) [inline src/main/java/TestFile2.java]: [Checkstyle] ERROR: Missing a Javadoc comment.

File: src/main/java/TestFile2.java (hunk 8 above)
Comment (sputnikci) [inline src/main/java/TestFile2.java]: [Checkstyle] ERROR: Missing a Javadoc comment.

File: src/main/java/TestFile2.java (hunk 9 above)
Comment (sputnikci) [inline src/main/java/TestFile2.java]: [Checkstyle] ERROR: Inner assignments should be avoided.

File: src/main/java/TestFile.java (hunk 1 above)
Comment (pjagielski) [inline src/main/java/TestFile.java]: Nice change

File: src/main/java/TestFile2.java (hunk 6 above)
Comment (sputnikci) [inline src/main/java/TestFile2.java]: [Checkstyle] ERROR: Missing a Javadoc comment.

File: src/main/java/TestFile2.java (hunk 7 above)
Comment (sputnikci) [inline src/main/java/TestFile2.java]: [Checkstyle] ERROR: Missing a Javadoc comment.

File: src/main/java/TestFile2.java (hunk 8 above)
Comment (sputnikci) [inline src/main/java/TestFile2.java]: [Checkstyle] ERROR: Missing a Javadoc comment.

File: src/main/java/TestFile2.java (hunk 9 above)
Comment (sputnikci) [inline src/main/java/TestFile2.java]: [Checkstyle] ERROR: Inner assignments should be avoided.

Comment (sputnikci) [inline src/main/java/TestFile2.java]: [FindBugs] WARNING: QBA: TestFile2.incorrectAssignmentInIfCondition() assigns boolean literal in boolean expression

File: src/main/java/TestFile.java (hunk 5 above)
Comment (sputnikci) [inline src/main/java/TestFile.java]: json

File: src/main/java/TestFile2.java (hunk 9 above)
Comment (sputnikci) [inline src/main/java/TestFile2.java]: [FindBugs] WARNING: QBA: TestFile2.incorrectAssignmentInIfCondition() assigns boolean literal in boolean expression

Comment (sputnikci) [inline src/main/java/TestFile2.java]: [FindBugs] WARNING: QBA: TestFile2.incorrectAssignmentInIfCondition() assigns boolean literal in boolean expression

File: src/main/java/TestFile.java (hunk 5 above)
Comment (sputnikci) [inline src/main/java/TestFile.java]: json

Comment (sputnikci) [inline src/main/java/TestFile.java]: json

File: src/main/java/TestFile2.java [hunk 10]
```diff
@@ -0,0 +1,16 @@
...
```
Comment (sputnikci) [inline src/main/java/TestFile2.java]: [FindBugs] WARNING: QBA: TestFile2.incorrectAssignmentInIfCondition() assigns boolean literal in boolean expression

File: src/main/java/TestFile2.java [hunk 11]
```diff
@@ -0,0 +1,16 @@
...
+            //do Something
+            value = false;
+        }
+        if (value = false) {
```
Comment (sputnikci) [inline src/main/java/TestFile2.java]: [FindBugs] WARNING: QBA: TestFile2.incorrectAssignmentInIfCondition() assigns boolean literal in boolean expression

File: src/main/java/TestFile2.java [hunk 12]
```diff
@@ -0,0 +1,16 @@
+class TestFile2 {
+    public String foo() {
+        return "bar";
+    }
+
+    private static void incorrectAssignmentInIfCondition() {
+        boolean value = false;
+        if (value == true) {
```
Comment (sputnikci) [inline src/main/java/TestFile2.java]: [Checkstyle] ERROR: Expression can be simplified.

File: src/main/java/TestFile2.java [hunk 13]
```diff
@@ -0,0 +1,16 @@
...
+            //do Something
+            value = false;
+        }
+        if (value == false) {
```
Comment (sputnikci) [inline src/main/java/TestFile2.java]: [Checkstyle] ERROR: Expression can be simplified.

File: src/main/java/TestFile.java (hunk 5 above)
Comment (sputnikci) [inline src/main/java/TestFile.java:10]: [Checkstyle] INFO: Line has trailing spaces.

**Pull request closed without merging**